tests/similarity/test_similarity.py

```diff
@@ -9,44 +9,44 @@
 def test_synset_similarity():
```

Comment:
I see that in some test there is a large difference in results

Reply:
If you mean the tolerance, when we try to identify the best similar word, for certain cases, we may have identified multiple words that have the same similarity score, if we do not preserve the order, the best candidate is chosen randomly. I have fixed a couple of places to fix the order, now we see more consistent results. If you mean the calculated similarity score, they vary for different methods.